# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dfb-linux-fusion/kernel-types.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./linux/drivers/char/fusion/types.h.orig	2000-01-01 00:00:00.000000000 +0100
+++ ./linux/drivers/char/fusion/types.h	2007-01-02 14:20:47.000000000 +0100
@@ -15,9 +15,12 @@
 #ifndef __FUSION__TYPES_H__
 #define __FUSION__TYPES_H__
 
+#include <linux/version.h>
+#if LINUX_VERSION_CODE < KERNEL_VERSION(2, 6, 19)
 typedef enum {
   false = 0,
   true  = !false
 } bool;
+#endif
 
 #endif
